pkgs/applications/misc/lutris/fhsenv.nix

```diff
@@ -121,4 +121,16 @@ in buildFHSUserEnv {
     ln -sf ${lutris-unwrapped}/share/applications $out/share
     ln -sf ${lutris-unwrapped}/share/icons $out/share
   '';
+
+  meta = {
+    inherit (lutris-unwrapped.meta)
+      homepage
+      description
+      platforms
+      license
+      maintainers
+      broken;
+
+    mainProgram = "lutris";
```

Comment:
Why not inherit everything and overwrite mainProgram?

Reply:
Lots of stuff in there that shouldn't be propagated to the wrapper.